app/main.py

```diff
@@ -1,2 +1,84 @@
+from __future__ import annotations
```

Comment:
Do you really need it? If yes, consider to update your python version. This is not a case for reject but still..